### DISCUSSION TITLE
Current project status and 3.0 release

### DISCUSSION THREAD

**Discussion by cipriancaba** [saleor/saleor#7371] [Q&A] [answered]
### What I'm trying to achieve
Thanks everyone here for doing a great job with saleor. We do appreciate all the hard work you are putting into the project

I am planning to release a new e-commerce platform. The platform has been in the works for most of the year under saleor version 3.0 under the assumption that this will be released in Q1. However, that wasn't the case and I there are some concerns that the May 2021 is not a realistic deadline either. We have made some updates to saleor under-the-hood to help our use-case and obviously an alpha release cadence is not helping us too much :)

### Describe a proposed solution
We are trying to get more visibility into the current planned release of saleor 3.0, so we can appropriately decide on what's the best plan of action. I saw a lot of new features being added, which is always great, and I am wondering if this is something that's going to continue for a while or we are mostly close to `beta`

### Other solutions I've tried and won't work
…

**Comment by patrys** [comment, marked as answer]
We are entering the final sprint before the API freeze, so expect a beta/release candidate in about two weeks. 3.0 is the biggest leap in Saleor's history, and it took a bit more time than anticipated but is absolutely worth it in terms of features 😊

You made the right call using 3.0 as the base for your site!

**Reply by cipriancaba** [reply]
Thanks again for the reassuring words and congrats on the great progress that saleor has achieved on all fronts. Looking forward to the release

**Reply by cipriancaba** [reply]
Hey @patrys . I see that the breaking changes still keep going. It might be worth having a pinned discussion regarding the roadmap and planned release date of the beta version (api freeze) where you keep us updated. I understand that maybe you need more time to finish everything you want in the 3.0 but we need some visibility into this so we can plan accordingly. 

Much appreciated

**Reply by patrys** [reply]
While it took longer than anticipated, we're merging the last breaking changes, and we are planning to publish a beta (API freeze) release next week.

**Reply by patrys** [reply]
@cipriancaba aaaand it's there! The first beta is out there for grabs. We're still working on compiling a usable release note summarizing the incredible amount of work that went into Saleor between versions 2.11 and 3.0.

**Reply by cipriancaba** [reply]
Amazing work 🥳

**Comment by weeping-somnambulist** [comment]
So are you just ignoring issues on 2.11 now? Lots of open issues without comment or resolution, and the saleor-platform is completely broken. Not a good look for someone trying out the platform. Whats going on with Saleor? API freeze on 3.0 was supposed to happen months ago, and still it's unusable.

**Reply by patrys** [reply]
I'm sorry to hear that Saleor does not work for you, but many companies are happy using Saleor. We have a wonderful team of great engineers working full time to bring you Saleor 3.0. So get off your high horse and be thankful for the hard work of the team and the community. You're getting it all for free.
